examples/dashboard-streamlit/README.md

```diff
@@ -0,0 +1,116 @@
+# Create A Dashboard on Saturn Cloud Using Streamlit
+
+![Streamlit logo](https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-resources/streamlit-logo.png "doc-image")
+
+## Overview
+With [Streamlit](https://streamlit.io/), data scientists can produce low-code data apps by abstracting away much of the technologies and protocols typically required for interactive data visualization. Deploying a Streamlit app on Saturn Cloud gives you not only a scalable backend but also a url for dissemination.
+
+In this example, we create a simple UI that shows a [Uniform Manifold Approximation and Projection (UMAP)](https://umap-learn.readthedocs.io/en/latest/) model projection of the famous MNIST digits and fashion datasets. The app will read the data, train the UMAP model, and produce a 3D graph of the result.
+
+## Creating the App
+All the app code is contained in a file called "app.py." To deploy this dashboard on Saturn Cloud, simply call **`streamlit run app.py --server.port 8000 --server.address 0.0.0.0`** as the command in a Saturn Cloud deployment. It is important to set the server port and address to `"8000"` and `"0.0.0.0"` so that Saturn Cloud can correctly deploy the app. See [Saturn Cloud docs](https://saturncloud.io/docs/examples/dashboards/dashboard/) for more detailed instructions on deploying this and other dashboards.
+
+### Import the Libraries
+
+This exercise uses Streamlit and UMAP to create a dashboard app:
+* [plotly](https://plotly.com/python/): interactive graphs
+* [streamlit](https://github.com/streamlit/streamlit): low-code data app framework
+* [UMAP](https://umap-learn.readthedocs.io/en/latest/): dimensionality reduction and visualization
```

Comment:
This feels a little redundant with the stuff above. I think if you mention plotly in the overview you could remove this